frozendict/__init__.py

```diff
@@ -1,6 +1,7 @@
 import collections
-import operator
-import functools
+
+
+iteritems = getattr(dict, 'iteritems', dict.items)
```

Comment:
it would be in scope of the `__hash__` fuction, so gc would sweep it soon after the interpreter leaves the function.